File: client/python/modcdp/BrowserbaseBrowserLauncher.py
from __future__ import annotations

from .BrowserLauncher import BrowserLaunchOptions, BrowserLauncher, LaunchedBrowser


class BrowserbaseBrowserLauncher(BrowserLauncher):
    def launch(self, options: BrowserLaunchOptions | None = None) -> LaunchedBrowser:
        raise NotImplementedError("launch.mode='bb' is not implemented yet.")

File: client/go/BrowserbaseBrowserLauncher.go
package modcdp

import "fmt"

type BrowserbaseBrowserLauncher struct {
	BrowserLauncher
}

func NewBrowserbaseBrowserLauncher(options LaunchOptions) BrowserbaseBrowserLauncher {
	return BrowserbaseBrowserLauncher{BrowserLauncher: NewBrowserLauncher(options)}
}

func (l BrowserbaseBrowserLauncher) Launch(options LaunchOptions) (*LaunchedBrowser, error) {
	return nil, fmt.Errorf("launch.mode=bb is not implemented yet")
}
